build.rs

```diff
@@ -524,6 +524,16 @@ fn obj_path(out_dir: &Path, src: &Path, obj_ext: &str) -> PathBuf {
     out_path
 }
 
+#[allow(box_pointers)]
```

Comment:
Let's remove the use of box pointers. We shouldn't need it for this.

Reply:
Please add a comment here explaining that this strips the build path for the purpose of reproducible builds.

Reply:
I removed the box pointers and I added a comment but I'm not super sure about the wording.